apps/publish/content/common.py

```diff
@@ -391,52 +250,17 @@ def _get_digital_id_for_package_item(self, package_item):
                 return package_item[config.ID_FIELD]
             return package_item_takes_package_id
 
-    def _get_subscribers_for_package_item(self, package_item):
-        """
-        Finds the list of subscribers for a given item in a package
-        :param package_item: item in a package
-        :return list: List of subscribers
-        :return string: Digital item id if there's one otherwise None
-        """
-        if package_item[ITEM_TYPE] not in [CONTENT_TYPE.TEXT, CONTENT_TYPE.PREFORMATTED]:
-            query = {'$and': [{'item_id': package_item[config.ID_FIELD]},
-                              {'publishing_action': package_item[ITEM_STATE]}]}
-        else:
-            package_item_takes_package = self.takes_package_service.get_take_package(package_item)
-            if not package_item_takes_package:
-                # this item has not been published to digital subscribers so
-                # the list of subscribers are empty
-                return []
-
-            query = {'$and': [{'item_id': package_item_takes_package[config.ID_FIELD]},
-                              {'publishing_action': package_item_takes_package[ITEM_STATE]}]}
-
-        return self._get_subscribers_for_previously_sent_items(query)
-
-    def _set_updates(self, original, updates, last_updated, preserve_state=False):
-        """
-        Sets config.VERSION, config.LAST_UPDATED, ITEM_STATE in updates document.
-        If item is being published and embargo is available then append Editorial Note with 'Embargoed'.
-
-        :param dict original: original document
-        :param dict updates: updates related to the original document
-        :param datetime last_updated: datetime of the updates.
-        :param preserve_state: doesn't update the state: it is used for preserving the state of digital package
-        """
-
-        if not preserve_state:
-            self.set_state(original, updates)
-
-        updates.setdefault(config.LAST_UPDATED, last_updated)
+    def _process_publish_updates(self, original, updates):
+        """ Common updates for published items """
+        desk = None
+        if original.get('task', {}).get('desk'):
+            desk = get_resource_service('desks').find_one(req=None, _id=original['task']['desk'])
+        if not original.get('ingest_provider'):
```

Comment:
Agree that it is part of the old code. 
But not sure we should change the source of the manually created articles on publishing action. Have to chat with POs.